code/modules/unit_tests/simple_animal_freeze.dm

```diff
@@ -302,6 +302,7 @@
 		/mob/living/simple_animal/slime,
 		/mob/living/simple_animal/slime/pet,
 		/mob/living/simple_animal/slime/random,
+		/mob/living/simple_animal/slime/random_tier_one,
```

Comment:
CC @tralezab we talked about this WRT the freeze test, on if you were okay freezing new subtypes of existing mobs or not. At the time you were not okay, thoughts today?